modules/client-configuration/pages/contact-methods-intro.adoc

```diff
@@ -2,15 +2,25 @@
 = Client Management Methods
 
 There are a number of ways that the {productname} Server can communicate with clients.
-Which one you use depends on the type of client, and your network architecture.
+Which one you use depends on the type of client, and your network architecture:
 
-The {productname} daemon ([command]``rhnsd``) runs on traditional client systems and periodically connects with {productname} to check for new updates and notifications.
-It does not apply to Salt clients.
-
-Push via SSH and Push via Salt SSH are used in environments where clients cannot reach the {productname} Server directly.
-In this environment, clients are located in a firewall-protected zone called a DMZ.
-No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
-
-OSAD is an alternative contact method between {productname} and traditional clients.
-OSAD allows traditional clients to execute scheduled actions immediately.
-It does not apply to Salt clients.
+Salt::
+is the default choice and recommended unless there are specific needs.
+For more information, see xref:contact-methods-salt.adoc[].
+Salt SSH::
+is useful only if network restrictions make it impossible for clients to establish contact to the server.
+This contact method has serious limitations.
+For more information, see xref:contact-methods-saltssh.adoc[].
+Traditional::
+is available for backwards compatibility only.
+This contact method has serious limitations.
+It does not scale as well as Salt.
++
+[WARNING]
+====
+Newer operating systems are not supported and will not be added in the future.
+Use it only when a needed feature is still not covered by Salt.
+For feature comparison, see xref:client-configuration:supported-features.adoc[].
+====
```

Comment:
I would even write that traditional is deprecated and will be removed in the next version